.github/workflows/update.yml

```diff
@@ -37,23 +38,18 @@ jobs:
           python -m pip install --upgrade pip
           pip install -r requirements.txt
 
-    - name: Install locales
+    - name: Clone previous cache data
       run: |
-        sudo apt-get install -yq tzdata locales
-        sudo locale-gen en_US.UTF-8
-        sudo locale-gen es_ES.UTF-8
-        locale -a
+        git clone https://github.com/open-covid-19/data --single-branch -b gh-pages ./output/public
 
-    - name: Update datasets
+    - name: Synchronize datasets
       working-directory: src
       run: |
-        echo ::set-env name=TIMESTAMP::"$(date +'%Y-%m-%d %H:%M')"
-        python ./update.py --verify simple --exclude weather,worldbank
```

Comment:
If update.py is no longer run here, where is it run?

Reply:
It runs on a schedule in a GCE instance that is managed by Google